### PR TITLE
Add expert cards to the entity details

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8555/

File: jujugui/static/gui/src/app/components/entity-details/content/content.js [hunk 1]
```diff
@@ -457,6 +458,21 @@ class EntityContent extends React.Component {
       </div>);
   }
 
+  /**
+    Generate the expert details
+  */
+  _generateExpert() {
+    const entityModel = this.props.entityModel;
+    // if (!this.props.hasPlans || entityModel.get('entityType') === 'bundle') {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-details/content/content.js]: Lets uncomment this and rely on this functionality

File: jujugui/static/gui/src/app/components/entity-details/entity-details.js [hunk 2]
```diff
@@ -136,7 +138,7 @@ class EntityDetails extends React.Component {
     Callback for when an entity has been successfully fetched. Though the
     data passed in is an Array of models, only the first model is used.
 
-    @method fetchSuccess
+    @method _fetchCallback
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-details/entity-details.js]: delete me 🤕

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8556/

File: jujugui/static/gui/src/app/components/expert-card/expert-card.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const GenericButton = require('../generic-button/generic-button');
+const SvgIcon = require('../svg-icon/svg-icon');
+
+const experts = {
+  spicule: {
+    supportedEntities: ['drillbit'],
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/expert-card/expert-card.js]: We aren't pre-defining this list. If the owner matches one of these top level keys and the charm has metrics and plans then it'll get the card.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8557/

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by huwshimi** [comment]
[Add expert cards to the entity details](https://trello.com/c/OFh0pktw/381-add-expert-cards-to-the-entity-details)

**Pull request merged**